### PR TITLE
Clarify docs on last 5 minutes query

### LINKED ISSUE #2459: TimeSeries Query for last 5 Minutes not Last 5 Minutes
The [usage docs](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/5a0e492b4098fe3c2df4d1821a432cbaee15edd6/docs/monitoring-usage.rst) say 

    query = client.query(METRIC, minutes=5)

will give you stats for the last five minutes. But since [the code](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/5a0e492b4098fe3c2df4d1821a432cbaee15edd6/monitoring/google/cloud/monitoring/query.py#L128) replaces seconds and microseconds with 0, it's not actually the last 5 minutes since you won't be querying any time interval between the end of the previous minute and the current second.

The reason this came up is in the system tests, I write a point, then try to query it back, but I never get the time series even if they are there, since I 

write_point: 2016-09-28T19:29:57.726637
query end_time:  2016-09-28T19:29:00.000000Z

so even though `write_point` happened first, it's end_time is after the end_time of the query.

Maybe people think second granularity doesn't matter? I'm curious why replacing the second/microseconds with 0 was done in the first place. I think at a minimum the docs are slightly misleading, since it's not the last 5 minutes in the colloquial sense (doesn't include the seconds up until now in the current minute).

cc @rimey @supriyagarg

### PR DESCRIPTION
**Pull request by waprin**
Fixes #2459 , cc @rimey @supriyagarg

### CODE REVIEW

File: docs/monitoring-usage.rst [hunk 1]
```diff
@@ -250,7 +250,7 @@ single time series. For this, you must have :mod:`pandas` installed;
 it is not a required dependency of ``google-cloud-python``.
 
 You can display CPU utilization across your GCE instances during
-the last five minutes as follows::
+the last five minutes, ending at the start of your current minute, as follows::
```
Comment (supriyagarg) [inline docs/monitoring-usage.rst]: your -> the

Comment (waprin) [inline docs/monitoring-usage.rst]: done

Comment (supriyagarg) [inline docs/monitoring-usage.rst]: Looking at it again, I would rewrite as follows. "last five minutes" is confusing in this context:

`You can display CPU utilization across your GCE instances over a five minute duration ending at the start of the current minute.`

Comment (waprin) [inline docs/monitoring-usage.rst]: agreed that's better, done

Comment (supriyagarg) [inline docs/monitoring-usage.rst]: Thanks. Looks okay, though a bit long. Sorry for editing my comment so many times. Perhaps @rimey can weigh in.

**Comment by rimey** [comment]
LGTM

**Pull request merged**